### PR TITLE
LoginPage-component-made

### LINKED ISSUE #12: Create a Login page and a Sign up page
- [ ] Login Page
- [ ] Sign Page
- [ ] Reset Password

Login page needs two components-
1. One for native login (email + password + remember me checkbox + reset password)
2. Second for google sign in

Signup page also needs two components-
1. One for native login (username + email + password + confirm password)
2. Second for google sign in

Reset Password page simply needs an email input field and submit button.

On checking "remember me" the users credentials should be stored in the local storage for a duration 1 week.
No need to create Submit Handlers for the forms right now

Both the pages need to be responsive and in accordance with ongoing color scheme and typography.
Take care to maintain consistency in both aspects as well as padding and margin.

For reference: 
- https://dribbble.com/shots/17408454-Login-Form-Prototype
- https://dribbble.com/shots/16316303-Login-and-Sign-up-Screens

Figma link - https://www.figma.com/file/BFLdR6rSd6rqjsx3nYepUT/UniShare?type=design&node-id=0%3A1&t=C0pM1FpOhI2hNqn7-1

### PR DESCRIPTION
**Pull request by shramanpaul**
@Anurag607 I have made the Login page I will further add the Sign In page and the localstorage functions to the mentioned.

fixes #12

### CODE REVIEW

**Comment by shramanpaul** [comment]
image

**Comment by sambhavgupta0705** [comment]
@shramanpaul I liked the design just change the background color gradient.
follow the website color scheme

**Comment by shramanpaul** [comment]
image

image
@Anurag607  @sambhavgupta0705  
which one should I use ?

**Comment by sambhavgupta0705** [comment]
Try using this hex color `#e6f3ff`

**Comment by shramanpaul** [comment]
Screenshot (5)
?

**Comment by sambhavgupta0705** [comment]
@Anurag607 for now we can go with the white background??

**Comment by sambhavgupta0705** [comment]
@shramanpaul we are only use tailwindcss in this project so you need to make your changes according to it

**Comment by shramanpaul** [comment]
ok @sambhavgupta0705

**Comment by Anurag607** [comment]
@shramanpaul you can refer to: [(https://gradienthunt.com/)] to get some good gradients.
make sure the color scheme of the site is yellow and blue them, so just make sure to maintain the color consistency.
or you can also add a background image if you want (you can find some in [https://unsplash.com/]), whichever looks good, just make sure to pick an img that matches with site i.e. should be related to resource sharing and is blue or yellow (or both) themed .


I like the design of the form, however, could we maybe have the logo of our institute a bit bigger it looks a bit small, also increase the size of the "Login" text in proportion to the logo. use monument font for it's font just add 'monu' to it's classname.

**Comment by shramanpaul** [comment]
@Anurag607 @sambhavgupta0705  how is the background image or should i change ?
Screenshot (6)

**Comment by shramanpaul** [comment]
@Anurag607  @sambhavgupta0705  How is the background-image or should i change ?
Screenshot (7)

**Comment by shramanpaul** [comment]
2.
Screenshot (7)

**Comment by shramanpaul** [comment]
@sambhavgupta0705 @Anurag607  i have updated the PR. Please review it.

**Comment by shramanpaul** [comment]
image

**Comment by Anurag607** [comment]
@shramanpaul any updates on the other pages?

**Comment by shramanpaul** [comment]
The Sign Up page is almost ready and I was leaning how to use and remove localstorage . @Anurag607

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./src/styles/LoginPage.css">
```
Comment (sambhavgupta0705) [inline index.html:7]: @shramanpaul can you explain addition of this line here

Comment (sambhavgupta0705) [inline index.html:7]: @shramanpaul can you explain addition of this line here

File: src/pages/LandingPage/index.jsx [hunk 2]
```diff
@@ -3,14 +3,18 @@ import NavBar from '../../components/NavBar'
 import HeroSection from '../../components/HeroSection'
 import Content from './content'
 import Footer from '../../components/Footer'
+import Login from '../login/index'
+
+
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/index.jsx:8]: remove these extra spaces

File: src/pages/LandingPage/index.jsx [hunk 3]
```diff
@@ -3,14 +3,18 @@ import NavBar from '../../components/NavBar'
...
 
 const LandingPage = () => {
   return (
     <>
-        <NavBar />
+        {/* <NavBar />
         <HeroSection />
         <Content />
-        <Footer />
+        <Footer /> */}
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/index.jsx:16]: same here

File: src/pages/LandingPage/index.jsx [hunk 4]
```diff
@@ -3,14 +3,18 @@ import NavBar from '../../components/NavBar'
...
 import Footer from '../../components/Footer'
+import Login from '../login/index'
+
+
 
 const LandingPage = () => {
   return (
     <>
-        <NavBar />
+        {/* <NavBar />
```
Comment (sambhavgupta0705) [inline src/pages/LandingPage/index.jsx:13]: don't close these components

File: src/pages/login/contentLogin.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <label className="text-gray-500">Password</label>
+            <input type="password" className="w-full border-b-2 border-gray-300 focus:border-blue-500 outline-none" required />
+          </div>
+          <div className="flex items-center mb-6">
+            <div className="flex-1 text-sm text-gray-600">
+              <a href="#" className="text-blue-500 hover:underline">Forgot Password?</a>
+            </div>
+            <div className="checkb1">
+              <input type="checkbox" id="remember" className="form-checkbox" />
+              <label htmlFor="remember" className="ml-2 text-sm text-gray-600">Remember Me</label>
```
Comment (sambhavgupta0705) [inline src/pages/login/contentLogin.jsx:38]: @Anurag607 should we add this remember me now as we need to enable local storage and cache for ir

**Review by sambhavgupta0705 — CHANGES_REQUESTED** [review]
Added some comments .

File: src/pages/login/contentLogin.jsx (hunk 5 above)
Comment (Anurag607) [reply src/pages/login/contentLogin.jsx:38]: Yes, we can also consider setting a defined expiry duration for it. Say around 1week?

Comment (sambhavgupta0705) [reply src/pages/login/contentLogin.jsx:38]: yeahh that sounds good

**Comment by Anurag607** [comment]
@shramanpaul any updates? Make sure to go through all the requested changes.

**Comment by shramanpaul** [comment]
> @shramanpaul any updates? Make sure to go through all the requested changes.

Ok @Anurag607

**Review by sambhavgupta0705 — COMMENTED** [review]
Great Work @shramanpaul 
Merging this PR for now,will contact you later for implementing features!!!

**Review by sambhavgupta0705 — APPROVED** [review]

**Pull request merged**